compiler-core/src/javascript/expression.rs

```diff
@@ -1485,8 +1518,8 @@ fn bit_array<'a>(
 
 #[derive(Debug)]
 struct SizedBitArraySegmentDetails<'a> {
-    has_explicit_size: bool,
     size: Document<'a>,
+    size_value: Option<BigInt>, // This is set when the segment's size is known at compile time
```

Comment:
Documentation should be on the line above and use `///` please

Reply:
Fixed.